### PR TITLE
Fixes the background SCP faction's description and changes earth to S

### PR DESCRIPTION
**Pull request by ghost**
…ite-53

## About the Pull Request

Quick PR (quick in simplicity not time it took me) that changes the Earth residence in the background menu to site-53 with appropriate descriptions like distance from urban populace, location and site director that are all redacted. The PR also fixes the SCP faction's description and removes the economic power entry of it.

## Why It's Good For The Game

Good for immersion

## Changelog

:cl: KirovClassBatlleCruiser, HaultyAnonie
fix: In the background menu, Earth is changed to Site-53 with an appropriate description.
tweak: The SCP foundation faction description in the background menu was improved and it does not have "economic power" anymore.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
also credits to anonhault or shoefluid however you want to call him

**Comment by Tsurupeta** [comment]
> also credits to anonhault or shoefluid however you want to call him

schizofield
I need to check if it borks saves or not, also this looks a little half-assed. If you're renaming factions, give singletons proper names too instead of just changing contents of it.

**Comment by ghost** [comment]
> > also credits to anonhault or shoefluid however you want to call him
> 
> schizofield I need to check if it borks saves or not, also this looks a little half-assed. If you're renaming factions, give singletons proper names too instead of just changing contents of it.

it shouldnt bork saves, didnt change var names or anything else to not bork saves

**Comment by Tsurupeta** [comment]
You don't know how saves work.

**Comment by ghost** [comment]
> You don't know how saves work.

correct

File: code/modules/culture_descriptor/location/locations_human.dm [hunk 1]
```diff
@@ -16,16 +16,10 @@
...
-	description = "Earth, home of humankind, has long been in a state of recovery following centuries of pollution and exploitation. Today, much of \
-	the Earth exists as a nature reserve, dedicated to preserving the history of humanity. The humans that remain live mostly in large urban zones \
-	across the globe. Huge, planned cities consisting of numerous arcologies with a focus on eco-friendliness. No longer the political power of \
-	Humanity, Earth is content to be something of a backwater and tourist destination for those looking to discover their heritage.<br><br>People from \
-	Earth are among the wealthier humans on average, though they are still comparable with those of Mars. Remarkably average, the people of Earth are \
-	much as they have been for hundreds of years, and are generally content with their lot in life. From a variety of ancient cultures, the people of \
-	Earth are as diverse as they have always been."
-	distance = "1AU"
-	capital = "Geneva"
-	economic_power = 1.2
```
Comment (TheDarkElites) [inline code/modules/culture_descriptor/location/locations_human.dm:28]: Have you checked what changing economic power does?

**Pull request closed without merging**